Vue CLI 2 & 3 Cypress Code Coverage Scripts (#51)
cypress-io/cypress-component-testing-examples: scripts/vue-cli-3/pr.md

1. Initialize a baseline project in the [${DIR}](.) subdirectory
   1. ${COMMAND_0}
   2. ${COMMAND_1}
   3. ${COMMIT_0}
2. Update app to move global styles into main.css file
   1. Move global app styles from [src/App.vue](src/App.vue) into [src/main.css](src/main.css)
   2. Update [src/main.js](src/main.js)
   3. ${COMMIT_1}
3. Add Cypress component testing support with sample tests
   1. ${COMMAND_2}
   2. Add [cypress.json](cypress.json)
   3. Add [cypress/plugins/index.js](cypress/plugins/index.js)

>    4. Add [src/App.spec.ct.js](src/App.spec.ct.js), [src/components/HelloWorld.spec.ct.js](src/components/HelloWorld.spec.ct.js), [src/Counter.vue](src/Counter.vue), [src/components/Counter.spec.ct.js](src/components/Counter.spec.ct.js)

Reviewer: Split into 2 lines?

```suggestion
   1. Update [src/App.vue](src/App.vue) to reference new [src/components/Counter.vue](src/components/Counter.vue) component
   1. Add [src/App.spec.ct.js](src/App.spec.ct.js), [src/components/HelloWorld.spec.ct.js](src/components/HelloWorld.spec.ct.js), [src/components/Counter.spec.ct.js](src/components/Counter.spec.ct.js) spec files
```

>    5. Add [src/CounterCompositionApi.vue](src/components/CounterCompositionApi.vue), a Composition API component and test in [src/components/CounterCompositionApi.spec.ct.js](src/components/CounterCompositionApi.spec.ct.js)

Reviewer: ```suggestion
   5. Add [src/CounterCompositionApi.vue](src/components/CounterCompositionApi.vue), a Composition API component with tests in [src/components/CounterCompositionApi.spec.ct.js](src/components/CounterCompositionApi.spec.ct.js)
```

   6. `npx cypress open-ct` (Notice that the fonts don't inherit global app styles)
   7. Edit [cypress/support/index.js](cypress/support/index.js) to import global app styles, the Cypress test preview should update automatically
   8. ${COMMIT_2}
4. Add Cypress Code Coverage
   1. ${COMMAND_3}
   2. Edit [cypress.json](cypress.json) to enable `coverage`
   3. Edit [cypress/plugins/index.js](cypress/plugins/index.js) to configure the Cypress code coverage task with Istanbul

>    4. Edit [cypress/support/commands.js](cypress/support/commands.js) to import Cypress code coverage support

Reviewer: ```suggestion
   4. Edit [cypress/support/index.js](cypress/support/index.js) to import Cypress code coverage support
```

   5. Update [babel.config.js](babel.config.js) to conditionally use `babel-plugin-istanbul` when Cypress Component tests are running

>    6. Add [.nycrc](.nycrc)

Reviewer: Same comment as the other readme

>    6. Add [.nycrc](.nycrc)

Author: ```suggestion
   6. Add [.nycrc](.nycrc)  to configure istanbul
```
